app/searches/lookup_categories.rb

```diff
@@ -16,8 +16,9 @@ def self.gene_names_in_category(category_name)
     DataModel::GeneClaimCategory
```

Comment:
Probably outside of the scope of this PR but this subroutine should retrieve gene categories directly.